### PR TITLE
[plugin.video.drnu] 5.0.12

### PR DESCRIPTION
**Pull request by msj33**
### Description
- Update of plugin.video.drnu to version 5.0.12
- Change of addon owner from twinther -> msj33/community based(Details here: https://github.com/xbmc-danish-addons/plugin.video.drnu/issues/20#issuecomment-460169727 )
### Checklist:
- [X] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [X] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-plugins/blob/master/CONTRIBUTING.md) document
- [X] Each add-on submission should be a single commit with using the following style: [plugin.video.foo] v1.0.0

### CODE REVIEW

**Comment by MartijnKaijser** [comment]
Just a notice, you shouldn't remove the copyright lines. Just add additional ones from future maintainers

**Comment by msj33** [comment]
Okay - Should I readd the lines and do another PR? Or is another commit sufficient.

And I can see that CI fails - But I know the codebase is working. Have not changed anything to the strings.xml (New requirements?)

This is my first PR for Kodi addons - So might needs some assisting here a bit :-)

File: plugin.video.drnu/addon.xml [hunk 1]
```diff
@@ -11,16 +11,16 @@
...
         <disclaimer lang="en">Some parts of this addon may not be legal in your country of residence - please check with your local laws.</disclaimer>
         <disclaimer lang="da">Nogle dele af denne addon er muligvis ikke lovlig i dit land - kontroller venligst dine lokale love.</disclaimer>
         <license>GPL 2.0</license>
         <platform>all</platform>
         <language>da</language>
-        <email>tommy@winther.nu</email>
-        <website>http://tommy.winther.nu</website>
-        <forum>http://tommy.winther.nu/wordpress/forums/forum/addons/dr-nu-player/</forum>
+        <email>msj33@hotmail.com</email>
+        <website></website>
```
Comment (Rechi) [inline plugin.video.drnu/addon.xml]: Website should list the site where the content is comming from.

File: plugin.video.drnu/clearfavorites.py [hunk 2]
```diff
@@ -1,7 +1,4 @@
 #
-#      Copyright (C) 2014 Tommy Winther
-#      http://tommy.winther.nu
-#
```
Comment (Rechi) [inline plugin.video.drnu/clearfavorites.py]: add the copyright info back

File: plugin.video.drnu/tvapi.py [hunk 3]
```diff
@@ -1,7 +1,4 @@
 #
-#      Copyright (C) 2014 Tommy Winther
-#      http://tommy.winther.nu
-#
```
Comment (Rechi) [inline plugin.video.drnu/tvapi.py]: add the copyright info back

File: plugin.video.drnu/tvgui.py [hunk 4]
```diff
@@ -1,7 +1,4 @@
 #
-#      Copyright (C) 2014 Tommy Winther
-#      http://tommy.winther.nu
-#
```
Comment (Rechi) [inline plugin.video.drnu/tvgui.py]: add the copyright info back

**Comment by Rechi** [comment]
There is no need to create a new PR just update the existing one.

You can find info about the new language format at https://kodi.wiki/view/Language_support#What_is_strings.po.

**Comment by msj33** [comment]
Wow...........my GIT commit apparently picked up some "changes" in another addon subdirectory I did not even touch when changing my addon files

Anyone got clues on what happened here or how I can fix it?

**Comment by msj33** [comment]
Closing this and trying from scratch again.

**Pull request closed without merging**